app/controllers/votes_controller.rb

```diff
@@ -1,8 +1,15 @@
 class VotesController < ApplicationController
-  def create
-    topic = Topic.find(params[:topic_id])
-    vote = topic.votes.build
-    vote.save!
-    redirect_to(topics_path)
-  end
+  # def create
+  #   topic = Topic.find(params[:topic_id])
+  #   vote = topic.votes.build
+  #   vote.save!
+  #   redirect_to(topics_path)
+  # end
+
+  # def update
+  #   topic = Topic.find(params[:topic_id])
+  #   vote = topic.votes.
+  #   vote.update!
+  #   redirect_to(topics_path)
+  # end
```

Comment:
Try to never commit commented code, it just sits there in the codebase and usually doesn't add value (you can always get stuff back via Git.)